internal/plans/planfile/planfile_test.go

```diff
@@ -167,3 +175,33 @@ func TestRoundtrip(t *testing.T) {
 		}
 	})
 }
+
+func TestWrappedError(t *testing.T) {
+	// Open something that isn't a cloud or local planfile: should error
+	wrongFile := "not a valid zip file"
```

Comment:
I ended up addressing this by combining _both_ errors, in the hope that at least one of them will hint at what the user did wrong. That said, because these are both meant as opaque formats, the exact details of the errors aren't really meant for normal users to think too hard about — they hopefully just point the way to remembering which file you _meant_ to click.